staging/src/kubevirt.io/client-go/kubecli/generated_mock_kubevirt.go

```diff
@@ -1497,8 +1497,8 @@ func (_m *MockVirtualMachineInstancePresetInterface) EXPECT() *_MockVirtualMachi
 	return _m.recorder
 }
 
-func (_m *MockVirtualMachineInstancePresetInterface) Get(ctx context.Context, name string, opts v12.GetOptions) (*v120.VirtualMachineInstancePreset, error) {
-	ret := _m.ctrl.Call(_m, "Get", ctx, name, opts)
+func (_m *MockVirtualMachineInstancePresetInterface) Get(ctx context.Context, name string, options v12.GetOptions) (*v120.VirtualMachineInstancePreset, error) {
+	ret := _m.ctrl.Call(_m, "Get", ctx, name, options)
```

Comment:
This change does not seem to belong in the `List` commit.

Reply:
Right! Rebase mess :)